packages/timeline-state-resolver-types/src/vizMSE.ts

```diff
@@ -51,6 +51,7 @@ export enum TimelineContentTypeVizMSE {
 	CONTINUE = 'continue',
 	LOAD_ALL_ELEMENTS = 'load_all_elements',
 	CLEAR_ALL_ELEMENTS = 'clear_all_elements',
+	SET_CONCEPT = 'set_concept',
```

Comment:
I think _set_, here and in the timeline object name, can be omitted for consistency with other objects. We use verbs mostly if the timeline object represents an irreversible fire-and-forget action that doesn't fit well in the state- and timeline-based paradigm, so _continue_, _load_all_, _clear_all_ are the exceptions here.
It is okay in command names, though, because they are discrete actions that the changes in state resolve to.

Reply:
Renamed it to "CONCEPT" in all cases except for the command